test/odbc/test_pg_datatype.cpp

```diff
@@ -0,0 +1,160 @@
+#include <gtest/gtest.h>
+#include <sqlext.h>
+#include "odbc_handler.h"
+#include "query_generator.h"
+
+using std::pair;
+
+class PSQL_DataTypes : public testing::Test{
+
+};
+
+TEST_F(PSQL_DataTypes, Datatypes_bigint) {
+
+  const string TABLE_NAME = "master_dbo.bigint_table";
+  const string COL1_NAME = "pk"; // primary key
+  const string COL2_NAME = "data";
+  const string DATATYPE_NAME = "sys.bigint";
+
+  const int BYTES_EXPECTED = 8;
+  const int NUM_ROWS = 4;
```

Comment:
Nit: could we group regular constants together and `EXPECTED` constants together? Just for organization & clarity's sake.

Reply:
Yeap we can do that.